ci/scripts/install_substrait_consumer.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+
+echo "Install Substrait Consumer Test Suite";
+
+git clone https://github.com/substrait-io/consumer-testing.git
+cd consumer-testing
+# avoid installing pyarrow
+grep -v pyarrow requirements.txt | while read line
```

Comment:
we should also ignore `--extra-index-url https://pypi.fury.io/arrow-nightlies --prefer-binary --pre`
```suggestion
grep -v 'pyarrow\|arrow-nightlies' requirements.txt | while read line
```
or just
```suggestion
grep -v arrow requirements.txt | while read line
```
 otherwise we get the following error when building:
```
#11 12.18 ERROR: You must give at least one requirement to install (see "pip help install")
#11 ERROR: executor failed running [/bin/bash -c -l /arrow/ci/scripts/install_substrait_consumer.sh]: exit code: 1
```